server/handlers.go

```diff
@@ -918,10 +957,16 @@ func (s *Server) getHandler(w http.ResponseWriter, r *http.Request) {
 		disposition = "attachment"
 	}
 
+	remainingDownloads := metadata.MaxDownloads - metadata.Downloads
```

Comment:
please, extract these three repeated lines from here https://github.com/dutchcoders/transfer.sh/pull/234/files#diff-105fedf355a9fcad82982745cef24f7aR914 in separate methods of `MetaData` encapsulating the logic